test/system/bans_test.rb

```diff
@@ -91,20 +94,35 @@ def test_banned_user_can_see_the_ban_notification_for_one_domain
 
   def test_banned_user_can_see_the_ban_notification_for_two_domains
     Ban.create!(user: @participant,
-                domain_name: 'temporary.test',
-                valid_from: Time.zone.today - 1, valid_until: Time.zone.today + 5)
+                domain_name: @valid_auction_with_offers.domain_name,
+                valid_from: Time.zone.today - DAYS_BEFORE, valid_until: Time.zone.today + DAYS_AFTER)
 
     sign_in_manually
 
     text = <<~TEXT.squish
       You are banned from participating in auctions for
-      domain(s): temporary.test, no-offers.test.
+      domain(s): no-offers.test, with-offers.test.
     TEXT
 
     visit auctions_path
     assert(page.has_css?('div.ban', text: text))
   end
 
+  def test_banned_user_can_see_the_ban_notification_only_for_active_auctions
+    Ban.create!(user: @participant,
+                domain_name: 'dummy-domain.net',
+                valid_from: Time.zone.today - DAYS_BEFORE, valid_until: Time.zone.today + DAYS_AFTER)
+
+    sign_in_manually
+
+    text = <<~TEXT.squish
```

Comment:
`text` is too broad. It is probably a notification? Or rather `expected_notification`? (if it's used only in `assert`).